apps/console/src/middleware.ts

```diff
@@ -1,16 +1,33 @@
+import createMiddleware from "next-easy-middlewares";
```

Comment:
@orangecoloured Could you add the reason why we use this lib?

To solve this kind of issue we can use the conditional middleware https://nextjs.org/docs/app/building-your-application/routing/middleware#conditional-statements To do additional work on the hub related logic. 

This lib's adaption rate worry me, normally I won't put library with such small participant into our codebase